src/session.js

```diff
@@ -475,6 +555,15 @@ ghostdriver.Session = function(desiredCapabilities) {
             }
         };
 
+        _decoratePromptBehavior(page);
+
+        // NOTE: The most common screen resolution used online is currently: 1366x768
+        // See http://gs.statcounter.com/#resolution-ww-monthly-201307-201312.
```

Comment:
will do.

does it sound reasonable to set the default to 1355x768 and maximize to 1920x1080?